src/anchor.js

```diff
@@ -28,7 +29,34 @@ const aggregators = {
             r += part.r
         }
         return new Point(x / len, y / len, r / len)
-    }
+    },
+    intersect: (config, name, parts) => {
+        const get_line_from_point = (point, offset=1000) => {
+            const p1 = [point.x, point.y]
+            const p2 = [point.x, point.y + offset]
+
+            let line = new m.paths.Line(p1, p2)
+            line = m.path.rotate(line, point.r, p1)
+
+            return line
+        }
+
+        a.unexpected(config, name, aggregator_common)
+        a.assert(parts.length==2, `Intersect expects exactly two parts, but it got ` + parts.length + `!`)
```

Comment:
This template string should probably be `Intersect expects exactly two parts, but it got ${parts.length}!` since the main reason for using them is that we can embed variables and not have to break and concatenate. :slightly_smiling_face: I'm nitpicking and I know it and I'm sorry - but maybe pointing even these small things out in the beginning will lead to that much smoother collab down the line :wink:

Reply:
Nit pickings are totally fine, and we should align to your preferred style rather the other way around. Made the change.